src/transformers/utils/quantization_config.py

```diff
@@ -1232,6 +1233,44 @@ def get_loading_attributes(self):
         return loading_attibutes_dict
 
 
+@dataclass
+class HiggsConfig(QuantizationConfigMixin):
+    """
+    HiggsConfig is a configuration class for quantization using the HIGGS method.
+
+    Args:
+        bits (int, *optional*, defaults to 4):
+            Number of bits to use for quantization. Default is 4.
+        p (int, *optional*, defaults to 2):
+            Parameter for the HIGGS quantization method. Default is 2.
```

Comment:
Could you explain a bit more what this is used for ?

Reply:
Added explanation